gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxList, NxTextLink, NxTable } from '@sonatype/react-shared-components';
+import {GalleryDescriptionTile} from '../../gallery-components/GalleryTiles';
+import NxListsExamples from './NxListExamples';
+
+const NxAlertPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>Lists take many forms:</NxP>
+      <NxList bulleted>
+        <NxList.Item>Simple data lists</NxList.Item>
+        <NxList.Item>Lists with clickable list items</NxList.Item>
+        <NxList.Item>Lists with bullets</NxList.Item>
+        <NxList.Item>Lists with actions</NxList.Item>
+        <NxList.Item>Lists with items that have multiple lines of text</NxList.Item>
+      </NxList>
+      <NxP>Lists can also have modified states depending on their content:</NxP>
+      <NxList bulleted>
+        <NxList.Item>Lists with no data</NxList.Item>
+        <NxList.Item>Error states</NxList.Item>
+      </NxList>
+      <NxP>
+        There are also lists that are "clickable". The list items in these lists indicate hover and click states and
+        when clicked, an event occurs - usually navigation. Clickable lists have hover and disabled states. They share
+        error and empty states with default lists.
+      </NxP>
+
+      <header className="nx-tile-subsection__title">
+        <h3 className="nx-h3">Description Lists</h3>
+      </header>
+      <NxP>
+        In addition to ordered and unordered lists, <NxCode>NxList</NxCode> also supports the
+        styling of description lists using <NxCode>NxList.DescriptionTerm</NxCode> and
+        <NxCode>NxList.Description</NxCode>. Terms and descriptions are laid out side-by-side
+        in rows. Currently only one description per term, and one term per description, are supported – not multiple.
+      </NxP>
+
+      <section className="nx-tile-subsection">
```

Comment:
Subsection header is missing.  Also be aware that there are convenience components for tile subsections.

Reply:
> Subsection header is missing. Also be aware that there are convenience components for tile subsections.

ef7e7f07d8f54b1ed605b492c6dc36366534b770